.gitignore

```diff
@@ -71,3 +71,7 @@ Carthage/
 
 # vsCode
 .vscode/
+
+# linaria
+
```

Comment:
the nittiest of nits, but can we ditch this extra line-break? 🙃

Reply:
Heck yes!